agriculture/static/js/dashboard.js

```diff
@@ -517,14 +520,41 @@ function updateWeatherStation(response) {
     if (controllerWindInfowElement != null)
         controllerWindInfowElement.innerText = weatherStationStatus[ID_WIND];
 
+    // Update the wind direction value.
+    controllerWindDir = weatherStationStatus[ID_WIND_DIR];
+    dir = "";
+    if (controllerWindDir = 0)
+        dir = "N"
+    if (controllerWindDir = 8)
+        dir = "NE"
+    if (controllerWindDir = 16)
+        dir = "E"
+    if (controllerWindDir = 24)
+        dir = "SE"
+    if (controllerWindDir = 32)
+        dir = "S"
+    if (controllerWindDir = 40)
+        dir = "SW"
+    if (controllerWindDir = 48)
+        dir = "W"
+    if (controllerWindDir = 56)
```

Comment:
If I remember correctly, this was fixed in a later commit, but just in case, use == to compare in an if block.